anchore_engine/configuration/localconfig.py

```diff
@@ -168,43 +181,48 @@ def load_defaults(configdir=None):
     return localconfig
 
 
-def load_policy_bundle_paths(src_dir=None):
+def load_policy_bundle_paths(src_dirs=None):
     global localconfig
 
     default_bundle_name = "anchore_default_bundle.json"
 
     # Get the dir containing policy bundles to put in the config
     policy_bundles_dir = localconfig["policy_bundles_dir"]
 
+    # This value will typically == None, outside of automated tests
+    if src_dirs == None:
+        src_dirs = policy_bundle_source_dirs()
+
     try:
-        if policy_bundles_dir is not None:
+        if (
+            src_dirs is not None
+            and len(src_dirs) > 0
+            and policy_bundles_dir is not None
+        ):
```

Comment:
in Python a `None` or an empty data structure like a list, will evaluate to `False` so this could be:
```suggestion
        if policy_bundles_dir is not None and src_dirs:
```

Reply:
In that case, couldn't it also further be simplified to `if policy_bundles_dir and src_dirs: ` ?

Reply:
I wasn't sure what `policy_bundles_dir` were (was not clear if that was a list as well). Sometimes the logic may want to really ensure it isn't a `None` so that is valid. If you know that it doesn't need to check for a `None` just if it is empty, then your suggestion works!